.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# General info
+<!--Delete or put N/A for any irrelevant sections-->
+
+**Issue number**: <!--Issue Number-->
+
+**Task description**: 
+
+ - Change 1
+ - Change 2
+ 
+<!-- What changed? What does this do? Provide screenshots if necessary--> 
+| Original   | Updated|
+|:---------------:|:---------------:|
+|<!--** original screenshot **--> | <!--** updated screenshot **-->|
+
+# Testing
+
+**Test coverage:** :
```

Comment:
Two `:` here, delete the one within the asterisks

Reply:
Fixed in 27db478